inspirehep/dojson/common/base.py

```diff
@@ -216,10 +217,13 @@ def collections(self, key, value):
 
     def get_value(value):
         primary = ''
-        if isinstance(value.get('a'), list):
+        if isinstance(value.get('a'), (list, tuple)):
             primary = value.get('a')[0]
+            if isinstance(primary, (list, tuple)):
+                primary = primary[0]
```

Comment:
Saw that these lines aren't covered in the coverage report. Is this case covered in the production data?